x-pack/plugins/cases/common/api/cases/case.ts

```diff
@@ -164,6 +164,36 @@ export const CasePostRequestRt = rt.intersection([
   }),
 ]);
 
+const CasesFindRequestSearchFieldsRt = rt.keyof({
+  'closed_by.username': null,
+  'closed_by.full_name': null,
+  'closed_by.email': null,
+  'closed_by.profile_uid': null,
+  'created_by.username': null,
+  'created_by.full_name': null,
+  'created_by.email': null,
+  'created_by.profile_uid': null,
+  description: null,
+  'connector.name': null,
+  'connector.type': null,
+  'connector.fields.key': null,
+  'connector.fields.value': null,
```

Comment:
Let's also remove them. It is also an implementation detail and we should not expose them.